epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyModel.java

```diff
@@ -123,8 +170,7 @@ public void bind(T view, List<Object> payloads) {
    *                             process, and follows the same general conditions for all
    *                             recyclerview change payloads.
    */
-  public void bind(T view, EpoxyModel<?> previouslyBoundModel) {
-    bind(view);
+  public void bindChanges(@NonNull T view, @NonNull EpoxyModel<?> previouslyBoundModel) {
```

Comment:
We can also leave the old name for backwards compatibility, but it wouldn't be as clear. Maybe we can get away with it since 2.0 was released only 5 days ago, but the few people who had time to upgrade would certainly not be amused.